test/flow/api/browser-upgrade.js

```diff
@@ -59,77 +61,67 @@ module.exports = function (browser) {
   browser.expect.map = () => browser.expect.element('div.leaflet-map-pane');
 
   browser.stopsTab = {};
-  browser.stopsTab.click = (done) => {
-    browser.click('.tabs-row .nearby-stops', () => {
-      done();
-    });
-  };
+  browser.stopsTab.click = () =>
+    browser
+      .waitForElementVisible('.tabs-row .nearby-stops', ELEMENT_VISIBLE_TIMEOUT)
+      .click('.tabs-row .nearby-stops');
 
   browser.back = {};
-  browser.back.click = (done) => {
-    browser.expect.element('.cursor-pointer.back').to.be.visible.before(ELEMENT_VISIBLE_TIMEOUT);
-    browser.click('.cursor-pointer.back', done);
-  };
+  browser.back.click = () =>
+    browser
+      .waitForElementVisible('.cursor-pointer.back', ELEMENT_VISIBLE_TIMEOUT)
+      .click('.cursor-pointer.back');
 
   browser.map = {
-    click(done) {
-      browser.click('div.map', done);
-    },
+    click: () =>
```

Comment:
Why remove the done, shouldn't that be usually passed around and called when finished?